src/CoreBundle/Event/HtmlIncludeEvent.php

```diff
@@ -58,13 +58,22 @@ public function getData()
         return $this->data;
     }
 
+    public function removeDataElement(string $key): void
+    {
+        unset($this->data[$key]);
+    }
+
+    public function updateDataElement(string $key, string $value): void
+    {
+        $this->data[$key] = $value;
+    }
+
     /**
      * @param mixed $value
      *
```

Comment:
If you remove the param here, the empty line in the comment can be removed as well.

Reply:
done

Reply:
done